### PR TITLE
Added reviewers()

### PR DESCRIPTION
**Pull request by neokd**
I have updated! Pls check

### CODE REVIEW

File: src/scrape_up/github/pull_request.py [hunk 1]
```diff
@@ -69,3 +69,20 @@ def files_changed(self):
         except:
             Message = "No files changed found"
             return Message
+
+    def reviewers(self):
+        """
```
Comment (nikhil25803) [inline src/scrape_up/github/pull_request.py:74]: Include this method in `documentation.md` as well

**Review by nikhil25803 — CHANGES_REQUESTED** [review]

File: src/scrape_up/github/pull_request.py (hunk 1 above)
Comment (neokd) [reply src/scrape_up/github/pull_request.py:74]: I just need to update the documentation ryt?

Comment (nikhil25803) [reply src/scrape_up/github/pull_request.py:74]: Yes.

**Comment by neokd** [comment]
Updated the documentation!

**Review by nikhil25803 — APPROVED** [review]

**Pull request merged**